Source/WebCore/rendering/RenderLayer.cpp

```diff
@@ -619,7 +619,7 @@ bool RenderLayer::shouldBeNormalFlowOnly() const
 
 bool RenderLayer::shouldBeCSSStackingContext() const
 {
-    return !renderer().style().hasAutoUsedZIndex() || renderer().shouldApplyLayoutContainment() || renderer().shouldApplyPaintContainment() || renderer().requiresRenderingConsolidationForViewTransition() || renderer().isRenderViewTransitionCapture() || renderer().isViewTransitionRoot() || isRenderViewLayer();
+    return !renderer().style().hasAutoUsedZIndex() || renderer().shouldApplyLayoutContainment() || renderer().shouldApplyPaintContainment() || renderer().requiresRenderingConsolidationForViewTransition() || renderer().isRenderViewTransitionCapture() || renderer().isViewTransitionContainingBlock() || isRenderViewLayer();
```

Comment:
Is this change correct? the spec says the ::view-transition pseudo should generate a stacking context

Reply:
I suspect the spec just wants the VT tree as a whole to be stacking (so that it can't interleave with anything else outside of it).

The spec does explicitly say the `::view-transition` pseudo though, and it could be observable (use author styles to remove other causes of being a stacking context, make a descendant a sibling in paint order, play with z-index).

I'll make them both stacking, which I think complies with the spec and works correctly.